### PR TITLE
CVE-2017-5093

### PR DESCRIPTION
**Pull request by cxs7700**
Set curated to true, half done

### CODE REVIEW

File: cves/CVE-2017-5093.yml [hunk 1]
```diff
@@ -34,35 +34,38 @@ description_instructions: |
...
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - commit: 0720b02e4f303ea6b114d4ae9453e3a7ff55f8dc
-  note: 
-- commit: 
-  note: 
+  note: |
+    Issue 2906133004. It was unable to be found through the command line, but was
```
Comment (CatherineOsadciw) [inline cves/CVE-2017-5093.yml]: Add a description of what the fix was

File: cves/CVE-2017-5093.yml [hunk 2]
```diff
@@ -34,35 +34,38 @@ description_instructions: |
...
+  note: |
+    Issue 2906133004. It was unable to be found through the command line, but was 
+    available on the Chromium site.
 vccs:
 - commit: 
-  note: 
-- commit: 
-  note: 
+  note: |
+    I was unable to find vulnerability contributing commits due to the commit
```
Comment (CatherineOsadciw) [inline cves/CVE-2017-5093.yml]: The automatically populated commit hash for the fix isn't always 100% accurate; try doing some digging through the git logs around that commit and through the Chromium bug report(s) for references to other commits.

File: cves/CVE-2017-5093.yml [hunk 3]
```diff
@@ -101,20 +106,22 @@ discovered:
...
-  automated: 
-  google: 
-  contest: 
+  answer: |
+    This was reported to the Chromium team by Luan Herrera and after doing some background
+    checks, I found out that Luan Herrera is a self-proclaimed security bug bounty hunter.
+  date: 2015-10-31
+  automated: false
+  google: false
+  contest: The $3000 bounty itself was the contest for Luan Herrera.
```
Comment (CatherineOsadciw) [inline cves/CVE-2017-5093.yml]: Contest should just be true or false, don't need a description

File: cves/CVE-2017-5093.yml [hunk 4]
```diff
@@ -124,12 +131,15 @@ interesting_commits:
...
-  answer:
+  answer: Yes
   commits:
-  - commit: 
-    note: 
-  - commit: 
-    note: 
+  - commit: 0720b02e4f303ea6b114d4ae9453e3a7ff55f8dc
+    note: |
+      It is interesting how even developers at top tech companies overlook validating
```
Comment (CatherineOsadciw) [inline cves/CVE-2017-5093.yml]: This is an observation that belongs in the conclusion; here you should instead provide a brief summary of what happened in that commit/why it was interesting.

File: cves/CVE-2017-5093.yml [hunk 5]
```diff
@@ -1,25 +1,25 @@
----
 CVE: CVE-2017-5093
 nickname: Don't Blink on Fullscreen
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: CWE-20 - Improper Input Validation
```
Comment (Jeremy523) [inline cves/CVE-2017-5093.yml]:
```suggestion
CWE: 20
```

File: cves/CVE-2017-5093.yml [hunk 6]
```diff
@@ -34,35 +34,38 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: |
-  UI spoofing in Blink. This vulnerability makes it possible to display a modal
+  UI spoofing in Blink. This vulnerability makes it possible to display a popup
```
Comment (Jeremy523) [inline cves/CVE-2017-5093.yml]: Can you give a brief overview of what Blink is or what it's responsible for?

File: cves/CVE-2017-5093.yml (hunk 2 above)
Comment (cxs7700) [reply cves/CVE-2017-5093.yml]: On the Chromium site, the commit hash is able to be found and matches the automatically populated description, so I went through with the rest of the process. However, `git show` and `git blame` do not work for the commit hash. Instead, I went onto the Chromium site and spent a considerable amount of time trying to find a VCC, and also read through many commits after seeing the logs for single files. Did some more digging just now, but throwing in the towel.

Comment (cxs7700) [reply cves/CVE-2017-5093.yml]: I found the section on the site that allows me to view the blames and diffs, and ultimately the VCC. Thanks for the feedback.

**Pull request merged**